src/ResourceManager/Storage/Commands.Management.Storage/help/Get-AzureRmStorageUsage.md

```diff
@@ -28,14 +28,22 @@ PS C:\>Get-AzureRmStorageUsage
 ```
 
 This command gets the Storage resources usage of the current subscription.
+ 
+
+### Example 2: Get the storage resources usage of specified location
+```
+PS C:\>Get-AzureRmStorageUsage -Location "US West"
+```
+
+This command gets the Storage resources usage of of specified location under the current subscription.
 
 ## PARAMETERS
 
 ### -DefaultProfile
 The credentials, account, tenant, and subscription used for communication with Azure.
 
 ```yaml
-Type: Microsoft.Azure.Commands.Common.Authentication.Abstractions.IAzureContextContainer
```

Comment:
@blueww you will need to run the `Update-MarkdownHelpModule` cmdlet again with the `-UseFullTypeName` in order to preserve the fully qualified name

Reply:
fixed.